recipe/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "cspy" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/torressa/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: a586f7f8a261770bfa0c06ad994e87911bac92200c418bfd48ba5d7a4dde1734
+  patches:
+    - patches/0001-unvendor-lemon-spdlog.patch
+    - patches/0002-fix-lemon.patch
+
+build:
+  number: 0
+  missing_dso_whitelist:
+    - '$RPATH/ld64.so.1'  # [s390x]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+  host:
+    - spdlog-fmt-embed
```

Comment:
Do we really need `spdlog-fmt-embed` or should we use `spdlog` instead?

Reply:
Yes, with only spdlog, there were some headers not found under a folder `spdlog/fmt/` (don't remember exactly) so  having a look at how `spdlog` is created, I went for the `fmt` version.

https://github.com/AnacondaRecipes/spdlog-feedstock/blob/ce0071cb793cfd6c85f53a912eadb9a51d786988/recipe/meta.yaml#L45